tests/integration/test_sanity.py

```diff
@@ -429,3 +429,24 @@ def test_tls_reconfig(cluster):
     commit_idx = cluster.leader_node().commit_index()
     cluster.node(3).wait_for_commit_index(commit_idx, gt_ok=True)
     cluster.node(3).wait_for_log_applied()
+
+
+@pytest.mark.skipif("not config.getoption('tls')")
+def test_tls_ca_cert_dir(cluster):
+    cluster.create(3, tls_ca_cert_location='dir')
+    assert cluster.execute('set', 'key', 'value')
+    assert cluster.execute('get', 'key') == b'value'
+
+
+@pytest.mark.skipif("not config.getoption('tls')")
+def test_tls_ca_cert_file(cluster):
+    cluster.create(3, tls_ca_cert_location='file')
+    assert cluster.execute('set', 'key', 'value')
+    assert cluster.execute('get', 'key') == b'value'
+
+
+@pytest.mark.skipif("not config.getoption('tls')")
+def test_tls_ca_cert_file(cluster):
```

Comment:
Should function name be different than above? It might be confusing if one fails other one succeeds, we may spend extra time to figure out which one failed. I'm surprised Python is okay with this :o